### ISSUE TITLE
Document how to bypass asset processing for CSS preprocessors

### ISSUE THREAD

**Issue by dannyskoog** [angular/angular#32811]
# 📚 Docs or angular.io bug report

### Description
With CLI we're able to configure a certain preprocessor to be used when processing styles. And within those styles we might make use of file relative paths as `background: url("../../foo.png")` or `background: url("./bar.png")`.

And as long as the asset is found at its given path at build time - we're fine, right? But let's say it's only guaranteed to be in place at runtime - then we'll end up with a compilation error. And after a few tries of trying to "shake" that file relative path to life you'll most likely end up browsing the Angular official docs for potential solutions.

Obviously using absolute paths is your friend, that's not the point. The point is that there actually is a way of pulling the previous paragraph off. Although, unfortunately, it's not documented. That is, if we're not treating the source code as pure documentation :)

So basically the caret sign (`^`) comes to the rescue and makes `background: url("^../../foo.png")` compile just perfectly fine. And the key to why this works lies within a few lines of code here https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/src/angular-cli-files/plugins/postcss-cli-resources.ts#L72. Effectively bypassing the processing (or existence validation if you will) of the asset.

So in short terms I think it would be valuable for CLI consumers to have this (tiny) information in the official docs. It will spare some lucky individuals a few hours of detective work :)

Thanks!

**Comment by kapunahelewong** [comment]
Hey @dannyskoog, thanks for this info. Would you be interested in submitting a PR to add it? I think this would be a nice addition to the documentation.

**Comment by dannyskoog** [comment]
Sure @kapunahelewong! I did some browsing around AIO to find a suitable spot for this particular documentation. What do you think about adding it to this section https://angular.io/guide/component-styles#non-css-style-files?

**Comment by kapunahelewong** [comment]
Seems like a good place to me. 😊Do you need help getting a PR together? I'm happy to help you get the PR through all the steps in the process. I also have [some instructions](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca) I share with people on working with aio in case you'd find it helpful.

**Comment by dannyskoog** [comment]
Great :) Those instructions seem more than enough in order for me to get a PR up.

So let’s take it from there for now. And thanks for your support!

**Comment by dannyskoog** [comment]
After info from @clydin it was brought to light that this is not a supported feature. Therefore I’ll close this issue for the time being.
